scripts/build_python.sh

```diff
@@ -51,6 +51,7 @@ declare -a extra_packages
 declare -a extra_gn_args
 declare chip_build_controller_dynamic_server=true
 declare enable_pw_rpc=false
+declare enable_webrtc=true
```

Comment:
I think it should be:
```suggestion
declare enable_webrtc=false
```

Reply:
That needs to be done by someone else. I did not pick this and if I change this, I may break something else. Right now, I want to scaffold to make both true/false work :)